src/xunit.netcore.extensions/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -28,17 +28,40 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
             Debug.Assert(ctorArgs.Count() >= 2);
 
             string issue = ctorArgs.First().ToString();
-            TestPlatforms platforms = (TestPlatforms)ctorArgs.Last();
+            TestPlatforms platforms = TestPlatforms.Any;
+            TargetFrameworkMonikers frameworks = (TargetFrameworkMonikers)~0;
+            
+            foreach (object arg in ctorArgs.Skip(1)) // First argument is the issue number.
+            {
+                if (arg is TestPlatforms)
+                {
+                    platforms = (TestPlatforms)arg;
+                }
+                else
+                {
+                    frameworks = (TargetFrameworkMonikers)arg;
```

Comment:
Just to be safe I would also check the type of the arg in this case as well and only cast if the type matches.